### ISSUE TITLE
Automate redemption and paging on Humble Bundle "Keys" pages

### ISSUE THREAD

**Issue by graingert** [Benjamin-Dobell/humble-bundle-key-redeemer#1]
Automatically page through and activate keys that are needed

**Comment by orta** [comment]
This seems to work for me?

<img width="824" alt="screen shot 2017-08-06 at 20 42 25" src="https://user-images.githubusercontent.com/49038/29008605-cdadf8b8-7ae7-11e7-8b36-1c1505f30532.png">

**Comment by Benjamin-Dobell** [comment]
Yes, it definitely works on those pages, that's the primary page I use.

However, the program itself intentionally only accesses browser contents in a read-only fashion i.e. it doesn't make any attempt to press buttons/links.

Technically there's nothing stopping us from automating that, I just felt more comfortable interacting with Humble Bundle myself rather than trying to automate it. For example, once you've pressed "click to redeem on Steam" you can no longer send a key as a gift through Humble Bundle - although you can of course still send the key manually.

Basically, this isn't a bug, it's just functionality that does not presently exist. I'm certainly open to feedback, so I'm happy to leave this issue open and see what other users think.

**Comment by orta** [comment]
I used this to register all the humble monthly bundle's I forgot about. thanks @Benjamin-Dobell  🍡
